### ISSUE TITLE
Add support for the dotnet-sonarscanner dotnet tool

### ISSUE THREAD

**Issue by Cheesebaron** [cake-contrib/Cake.Sonar#114]
Would be really nice to switch over to use the dotnet-sonarscanner tool, such that .NET Core and .NET 5.0+ projects also can be scanned using this plugin.

**Comment by HofmeisterAn** [comment]
Indeed. In the meantime you can use following approach to use the official scanner: https://github.com/cake-contrib/Cake.Sonar/issues/96#issuecomment-779401679.

**Comment by Cheesebaron** [comment]
> Indeed. In the meantime you can use following approach to use the official scanner: [#96 (comment)](https://github.com/cake-contrib/Cake.Sonar/issues/96#issuecomment-779401679).

Neat
